lib/mllparty/time_utils.ex

```diff
@@ -0,0 +1,31 @@
+defmodule TimeUtils do
+  def format_seconds(seconds) do
+    case seconds do
+      s when s < 60 ->
+        "#{s} second#{pluralize(s)}"
+
+      s when s < 3600 ->
+        minutes = div(s, 60)
+        seconds = rem(s, 60)
+        "#{minutes} minute#{pluralize(minutes)}, #{seconds} second#{pluralize(seconds)}"
+
+      s when s < 86400 ->
```

Comment:
Nit, but I usually like to make "magic values" named module level variables for readability  e.g.:
```elixir
defmodule TimeUtils do
  @one_minute 60
  @one_hour @one_minute * 60
  @one_day @one_hour * 24
   
  def format_seconds(seconds) do
    case seconds do
      s when s < @one_minute ->
         ...
     end
  end
end
```

Reply:
Good suggestion